src/transformers/utils/import_utils.py

```diff
@@ -386,6 +394,10 @@ def is_torch_fx_available():
     return _torch_fx_available
 
 
+def is_bs4_available():
+    return _bs4_available
```

Comment:
```suggestion
    return importlib.util.find_spec("bs4") is not None
```
Just do this and remove line 128 to 133 IMO.